### DISCUSSION TITLE
How to convert a Detectron 2 model to onnx?

### DISCUSSION THREAD

**Discussion by EiriniKot** [onnx/onnx#4610] [Q&A]
I am trying to convert a detectron 2 model to onnx using 

torch.onnx.export

I have error (TypeError: string indices must be integers) regarding the args given to export

**Comment by jcwchen** [comment]
Hi @EiriniKot,
It's more a question for the exporter (torch.onnx.export). Please raise this question in their repo and mention ONNX in the title to get the best help from the exporter experts.
